future-apps/payroll/test/contracts/Payroll_payday.test.js

```diff
@@ -301,20 +309,13 @@ contract('Payroll payday', ([owner, employee, anyone]) => {
 
                 context('when the requested amount is lower than the total owed salary', () => {
                   context('when the requested amount represents less than a second of the earnings', () => {
-                    const requestedAmount = salary.div(2)
+                    const requestedAmount = salary.div(2).floor()
```

Comment:
This is what I noticed yesterday, after turning the base salary to 1 USD per second, this became flaky